packages/apps/auth0-actions/tests/redirect_to_full_registration.test.ts

```diff
@@ -119,8 +118,8 @@ describe('redirect_to_full_registration', () => {
         request: { hostname: 'stage.wellcomecollection.org' } as EventRequest,
         secrets: auth0SecretsObject as EventSecrets,
       } as Event<Auth0User>);
-    const envUrl = event.secrets.AUTH0_ACTION_URL_STAGE;
-    const successUrl = `https://${envUrl}/success`;
+
+    const successUrl = `https://${event.secrets.IDENTITY_APP_BASEURL}/success`;
```

Comment:
Why do we need to prepend the https here but not on line 60?

Reply:
good catch! We don't, we just need `/success' - fix incoming 👍